core/src/main/java/io/questdb/cairo/TableReader.java

```diff
@@ -670,6 +670,11 @@ private void closeDeletedPartition(int partitionIndex) {
         }
         int baseIndex = getPrimaryColumnIndex(columnBase, 0);
         int newBaseIndex = getPrimaryColumnIndex(getColumnBase(partitionIndex + 1), 0);
+        for (int i = baseIndex; i < newBaseIndex - 1; i++) {
```

Comment:
nit: `newBaseIndex` could have  value - 1 assigned to it in the first place rather than calculating it in a loop

Reply:
ok, done